server/models/room.json

```diff
@@ -26,7 +26,7 @@
     "minimum_capacity": {
       "type": "number",
       "requied": true,
-      "default": 0
+      "default": 1
```

Comment:
Try setting `default: 0` back and try if we still get the negativeError. thanks!

Reply:
🤠👉🏻